api/lib/infrastructure/serializers/jsonapi/campaign-participation-overview-serializer.js

```diff
@@ -0,0 +1,30 @@
+const _ = require('lodash');
+const { Serializer, Deserializer } = require('jsonapi-serializer');
+
+const CampaignParticipation = require('../../../domain/models/CampaignParticipation');
+
+module.exports = {
+
+  serializeForPaginatedList(userCampaignParticipationOverview) {
+    const { campaignParticipationOverviews, pagination } = userCampaignParticipationOverview;
+    return this.serialize(campaignParticipationOverviews, pagination);
+  },
+
+  serialize(campaignParticipationOverview, meta) {
+    return new Serializer('campaign-participation-overview',
+      {
+        attributes: ['isShared', 'sharedAt', 'createdAt', 'organizationName', 'assessmentState', 'campaignCode', 'campaignTitle'],
+        meta,
+      }).serialize(campaignParticipationOverview);
+  },
```

Comment:
Question : est-ce qu'on a besoin de ces deux méthodes, est-ce que : 

```suggestion
  serialize(userCampaignParticipationOverview) {
      const { campaignParticipationOverviews, pagination } = userCampaignParticipationOverview;

    return new Serializer('campaign-participation-overview',
      {
        attributes: ['isShared', 'sharedAt', 'createdAt', 'organizationName', 'assessmentState', 'campaignCode', 'campaignTitle'],
        meta : pagination,
      }).serialize(campaignParticipationOverview);
  },
```

ne serait pas fonctionnel ? (même avec le meta vide ? )

Reply:
On a repris la même logique qu'ailleurs, en l'occurence: https://github.com/1024pix/pix/blob/8f5fa350bea61f67e4a83c7c90e49b06f206965f/api/lib/infrastructure/serializers/jsonapi/jury-session-serializer.js#L5-L10